Source/gs/GSH_Vulkan/GSH_VulkanTransferHost.cpp

```diff
@@ -13,6 +13,8 @@ using namespace GSH_Vulkan;
 #define DESCRIPTOR_LOCATION_MEMORY 0
 #define DESCRIPTOR_LOCATION_XFERBUFFER 1
 #define DESCRIPTOR_LOCATION_SWIZZLETABLE_DST 2
+#define DESCRIPTOR_LOCATION_MEMORY_8BIT 3
+#define DESCRIPTOR_LOCATION_MEMORY_16BIT 3
```

Comment:
`DESCRIPTOR_LOCATION_MEMORY_16BIT` needs to be 4 otherwise Metal complains that it can't generate a layout for the shader.